[ADD] Payroll: Time Off To Report (#14501)
odoo/documentation: content/applications/hr/payroll/time_off_to_report.rst

==================
Time off to report

> ==================

Reviewer: Hi Lara! More lengthy of a comment here (: 

Since this is a shorter doc, I'm wondering if you'd like to experiment with our new action-focused document type and rewrite this in a new style. Currently, the doc is clear in explaining the dashboard, context, and examples, but if this document is meant to be read for only one reason (resolve payroll discrepancies), we can shift this doc to be more about what the user needs to do and why.

So instead of titling this "Tiome off to report", for instance, we can title it something like "Resolve time off conflicts". 

In addition, I have some questions for you to consider to help reframe this doc in a more action-oriented way:
- Who is the intended user for this doc (payroll administrators, HR officers, or general employees)?
- What is the specific goal this user is trying to accomplish when they open this page?
- At what point in their workflow would the payroll admin land here? Can we structure the doc so they can get from “problem identified” → “steps to fix it” quickly?
- Can we reframe the introduction: instead of explaining payroll theory, can we start with the problem the user faces (e.g., “An employee’s time off was approved after payroll was processed—what do you do?”)?
- What is the main decision point for the user—defer multiple entries at once, or handle them one by one?
- Would it help to start each section title with a verb (e.g., “Defer multiple time off entries” → “Defer multiple entries at once”)?

> It is not uncommon for payroll to be processed a few days before the end of a pay period, so that

Reviewer 2: ```suggestion
Payroll is often processed a few days before the end of a pay period, so that
```

   Instead of cancelling the paycheck and reissuing it, causing delays for the employee and more
   work for the payroll department, Odoo allows for the deferral of the time off taken, to the
   following pay period.

   This ensures all time off balances are correct, and the employee is properly compensated for
   their time.

.. _payroll/time-off-dashboard:

Time off dashboard
==================

When time off requests are submitted for a time period that was already processed on a payslip, the
time off requests appear in the *Time Off* page of the **Payroll** app. To access this, navigate to
:menuselection:`Payroll app --> Work Entries --> Time Off to Report`.

The :guilabel:`Time Off` page default filter is :guilabel:`To Defer`, and displays all requests with
a :guilabel:`Payslip State` of :guilabel:`To defer to next payslip`. This is because the employee
was *already paid* for that time as worked time, and it was logged as regular time spent at work.

.. image:: time_off_to_report/time-off-to-report.png
   :alt: A list of all time off requests that were not approved before payslips were generated.

Defer multiple time off entries
===============================

> In order to keep the employee's time off balances correct, the time off request **must** be applied

Reviewer 2: ```suggestion
To keep the employee's time off balances correct, the time off request **must** be applied
```
